### PR TITLE
fix: konflux: update secret name

### PR DESCRIPTION
**Pull request by psturc**
the secret should not have such generic name as "ci-secrets". renaming to "konflux-ci-secrets" to easily distinguish e.g. from rhtap secrets

based on [this slack conversation](https://redhat-internal.slack.com/archives/CBN38N3MW/p1706875012741849) with dptp team

### CODE REVIEW

**Comment by psturc** [comment]
/pj-rehearse pull-ci-redhat-appstudio-image-controller-main-image-controller-e2e

File: ci-operator/step-registry/redhat-appstudio/rhtap-installer/e2e-test/redhat-appstudio-rhtap-installer-e2e-test-ref.yaml [hunk 1]
```diff
@@ -3,7 +3,7 @@ ref:
   cli: latest
   commands: redhat-appstudio-rhtap-installer-e2e-test-commands.sh
   credentials:
-  - mount_path: /usr/local/ci-secrets/redhat-appstudio-qe
+  - mount_path: /usr/local/ci-secrets/rhtap
```
Comment (psturc) [inline ci-operator/step-registry/redhat-appstudio/rhtap-installer/e2e-test/redhat-appstudio-rhtap-installer-e2e-test-ref.yaml:6]: this was a copy-paste issue, the location of a secret for rhtap jobs should be `rhtap`

**Review by elsony — APPROVED** [review]
/lgtm

**Comment by psturc** [comment]
/pj-rehearse pull-ci-redhat-appstudio-infra-deployments-main-appstudio-hac-e2e-tests pull-ci-redhat-appstudio-infra-deployments-main-appstudio-e2e-tests pull-ci-redhat-appstudio-infra-deployments-main-appstudio-upgrade-tests pull-ci-redhat-appstudio-infra-deployments-main-appstudio-load-test

**Pull request merged**